libnmstate/nm/active_connection.py

```diff
@@ -199,17 +200,16 @@ def remove_handlers(self):
         self.handlers = set()
         for handler_id in self.device_handlers:
             self.nmdevice.handler_disconnect(handler_id)
-        self.device_handlers = set()
 
 
 def _is_device_master_type(nmdev):
     if nmdev:
-        gobject = nmclient.GObject
+        gobject = GObject
         is_master_type = (
-            gobject.type_is_a(nmdev, nmclient.NM.DeviceBond)
-            or gobject.type_is_a(nmdev, nmclient.NM.DeviceBridge)
-            or gobject.type_is_a(nmdev, nmclient.NM.DeviceTeam)
-            or gobject.type_is_a(nmdev, nmclient.NM.DeviceOvsBridge)
+            gobject.type_is_a(nmdev, NM.DeviceBond)
+            or gobject.type_is_a(nmdev, NM.DeviceBridge)
+            or gobject.type_is_a(nmdev, NM.DeviceTeam)
+            or gobject.type_is_a(nmdev, NM.DeviceOvsBridge)
```

Comment:
What I mean above is `GObject.type_is_a(..)` instead of `gobject.type_is_a(..)`, just to clarify.